src/support/file.h

```diff
@@ -28,16 +28,28 @@
 #include <vector>
 
 namespace wasm {
+
+namespace Flags {
+  enum BinaryOption {
+    Binary,
+    Text
+  };
+  enum DebugOption {
+    Debug,
+    Release
+  };
+}
```

Comment:
How would accessing the values look?

The issue is that e.g. Binary is the name of something else as well, so it needs `Flags::` or some other prefix. And `Flags::` seems ok to me.